client/src/components/pages/Signup/index.js

```diff
@@ -205,105 +210,103 @@ export default class signup extends Component {
                   className="signup__input"
                   label="username"
                   placeholder="Username"
-                  value={username.value}
+                  value={username}
                   onChange={this.handleChange}
                 />
+                {usernameError && (
+                  <p className="error-message">{usernameError}</p>
+                )}
                 <Input
                   type="email"
                   name="email"
                   className="signup__input"
                   label="email"
                   placeholder="Email"
-                  value={email.value}
+                  value={email}
                   onChange={this.handleChange}
                 />
+                {emailError && <p className="error-message">{emailError}</p>}
```

Comment:
instead of the checking for the error msg here 
you can take a look at `Input` common component so you can pass the error like this : 
```js
 errMsg={errors.email}
```

Reply:
the span for the errMsg at the `Input` common component need style fixing